optimade/validator/utils.py

```diff
@@ -247,6 +247,10 @@ def get(self, request: str):
             except requests.exceptions.ConnectionError as exc:
                 errors.append(str(exc))
 
+            # Read timeouts should prevent further retries
+            except requests.exceptions.ReadTimeout as exc:
+                raise ResponseError(str(exc)) from None
```

Comment:
```suggestion
                raise ResponseError(str(exc)) from exc
```
Right?

Reply:
Either works, yeah, this is just to suppress the traceback from requests->urllib->CPython code. `from exc` is probably cleaner.